pkg/scheduler/extender/extender.go

```diff
@@ -250,6 +250,21 @@ func (e *Extender) gatherCapacityRequestsByProvisioner(ctx context.Context, pod
 			}
 			continue
 		}
+		if v.Ephemeral != nil {
+			claimSpec := v.Ephemeral.VolumeClaimTemplate.Spec
+			if storageType, ok := scs[*claimSpec.StorageClassName]; ok {
+				storageReq, ok := claimSpec.Resources.Requests[coreV1.ResourceStorage]
+				if !ok {
+					ll.Errorf("There is no key for storage resource for PVC %s", v.Name)
```

Comment:
Does it mean that incorrect SC name was passed?

Reply:
No, it means that Volume size is not set, but from k8s specification it always must not be empty. Just check to not raise panic here, if we received incorrect data